src/test/java/guitests/FilterTests.java

```diff
@@ -64,6 +64,91 @@ public void milestoneAliasFilterTest() {
         assertEquals(0, issuePanel.getIssueCount());
     }
 
+    @Test
+    public void countFilterTest(){
+        ListPanel issuePanel = find("#dummy/dummy_col0");
+
+        // Checking 2 issues show for count:2
+        click("#dummy/dummy_col0_filterTextField");
+        selectAll();
+        type("count:2");
+        push(KeyCode.ENTER);
+
+        assertEquals(2, issuePanel.getIssueCount());
+
+        // Checking 7 issues shown for count:7
+        click("#dummy/dummy_col0_filterTextField");
+        selectAll();
+        type("count:7");
+        push(KeyCode.ENTER);
+
+        assertEquals(7, issuePanel.getIssueCount());
```

Comment:
Nope, 0 and 10 are boundary values. Both have been tested. In addition, 2 and 7 are being tested as well.